RELEASE_NOTES.md

```diff
@@ -1,5 +1,79 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Infra Client 15.7
+
+## Updated Resources
+
+### archive_file
+
+The `archive_file` will now only change ownership on files and directories that were part of the archive itself. This is important so you don't change permissions on important high level directories such as /etc or /bin just by extracting a file into those directories. Thanks for this fix [@bobchaos](https://github.com/bobchaos/)
+
+### cron and cron_d
+
+The `cron` and `cron_d` resources now include a `timeout` property, which allows you to configure actions to perform when a job times out. This property accepts a hash of timeout configuration options:
+
+- `preserve-status`: true/false with a default of false
```

Comment:
@tas50 Are these bullet points supposed to have "true/false" mentions formatted a la `true` and `false`?

Reply:
It seemed like a lot of formatting there if I did it that way

Reply:
I'm trying to be consistent as you did use `true` and `false` further down in this batch of release notes...

Reply:
Fixed